dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/page/HTMLPageAssetRenderedBuilder.java

```diff
@@ -186,16 +186,21 @@ public PageView build(final boolean rendered, final PageMode mode) throws DotDat
         return Optional.ofNullable(contentlet);
```

Comment:
In general this Builder IMO looks such as API or Service, it very smart to be just a builder to create a bean. It is always confusing